README.md

```diff
@@ -89,7 +117,7 @@ Establish a connection to the given peer. You usually won't need to use this fun
      - `client`. Boolean. If true, the connection was initiated by this node.
      - `peer`. Object describing the peer. (Will be the same object that was passed into this method.)
 
-#### `net.on('connection', (socket, details) => {})`
+#### `swarm.on('connection', (socket, details) => {})`
```

Comment:
we should document the api for details

Reply:
yep - are we cool to do that in the next pass of the readme (during the diagramming phase)?

Reply:
Yes sounds good.